bin/middlewares/map_style.js

```diff
@@ -0,0 +1,40 @@
+const styleBuilder = require('@qwant/map-style-builder')
+const styleConfigure = require('mapbox_style_configure')
+const qwantStyle = require('@qwant/qwant-basic-gl-style/style.json')
+const path = require('path')
+
+
+module.exports = function (config, languages) {
+
+  /* pre-build style on server start */
+  let options = {
+    output: 'production', // 'debug' | 'production' | 'omt'
+    outPath : __dirname + '/../public/mapstyle',
+    i18n : true,
+    icons : true,
+    pixelRatios : [1,2],
+    styleDir : path.resolve(path.join(__dirname, '..', '..', 'node_modules', '@qwant' , 'qwant-basic-gl-style')),
+    conf : {tileserver_base : '{tileserver_base}', tileserver_poi : '{tileserver_poi}', spriteserver : '{spriteserver}', fontserver : '{fontserver}'},
+    ignoreSprite : true
+  }
+
+  let builtStyle = styleBuilder(qwantStyle, options)
+
+  return function (req, res) {
+    /* get lang routine */
+    let langParameter = req.query.lang
```

Comment:
At this point, it could be a good idea to add documentation about this kind of endpoints and parameters. In the readme for example.  
Especially as this one will be used by other services.

Reply:
compression (the most used middleare) enable gzip for all request, it take a filter param but i think it's dirty way. 
or i can enable compression for all request

Reply:
done

Reply:
could we describe the default behavior (without `lang` too) ?

Reply:
yep added default param to doc